client/components/Board.js

```diff
@@ -14,17 +14,22 @@ class Board extends Component {
   }
 
   render () {
+    var counter = 0
+    var levelTheme = this.props.currentLevel < 5 ? 'basement' : 'cargo'
+    console.log('the level ', levelTheme)
     return (
       <div id='board'>
         <div className='board-container'>
           {
             this.props.board.map((row, rowIndex) => {
               return row.map((col, colIndex) => {
+                let oddEven = counter % 2 == 1 ? 'odd' : 'even'
+                counter++
                 return (
                 col == 2 ? 
-                <div key={parseInt(rowIndex.toString() + colIndex.toString())} className='tile' ref={rowIndex.toString() + colIndex.toString()}> <img src='/resources/images/box-tile.svg' className='box-tile' /> </div>  :
-                 col == 1 ?  <div key={parseInt(rowIndex.toString() + colIndex.toString())} className='tile' ref={rowIndex.toString() + colIndex.toString()}> <div className='elevator-bottom'> <img src='/resources/images/elevator-top.svg' className='elevator-tile' /> </div> </div> :
-                  <div key={parseInt(rowIndex.toString() + colIndex.toString())} className='tile' ref={rowIndex.toString() + colIndex.toString()}></div>
+                <div key={parseInt(rowIndex.toString() + colIndex.toString())} className={'tile ' + levelTheme + ' ' + oddEven} ref={rowIndex.toString() + colIndex.toString()}> <img src='/resources/images/box-tile.svg' className='box-tile' /> </div>  :
```

Comment:
I gave this a go but just caused things to break, ill just be doing it wrong.. going home now so did you want to refactor it real quick? otherwise its working in this way